website/snippets/_sl-dbt-mesh-faq.md

```diff
@@ -0,0 +1,30 @@
+When using the dbt Semantic Layer in a [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) setting, we recommend the following:
+
+- You have one global project that contains your Semantic Layer configurations.
+- Then as you build your Semantic Layer, you can cross reference dbt models across your various projects or packages to create your semantic models using the [two-argument `ref` function](/reference/dbt-jinja-functions/ref#ref-project-specific-models)( `ref('project_name', 'model_name')`).
```

Comment:
I recommend also linking to:
- https://docs.getdbt.com/docs/collaborate/govern/project-dependencies